examples/pipelines/hf_dataset_pipeline/README.md

```diff
@@ -0,0 +1,63 @@
+In this guide, we explain how to implement a KubeFlow pipeline and run it on KubeFlow on GKE (Google Kubernetes Engine).
+
+# 1. Implement the components
+
+1. Create a "components" folder where all KubeFlow components will live.
```

Comment:
I think a lot of this documentation is repeated from something i'v already included before and seems to be more general to kubeflow rather than this specific pipeline.

Could we:

* Define this somewhere else? Maybe on the root level of the pipelines.
* Combine it with the current existing documentation?https://github.com/ml6team/express/blob/main/docs/kubeflow_101.md